src/sonic-bgpcfgd/bgpcfgd/managers_bbr.py

```diff
@@ -0,0 +1,120 @@
+from swsscommon import swsscommon
+
+from .log import log_err, log_info, log_crit
+from .manager import Manager
+from .utils import run_command
+
+
+class BBRMgr(Manager):
+    """ This class initialize "BBR" feature for  """
+    def __init__(self, common_objs, db, table):
+        """
+        Initialize the object
+        :param common_objs: common object dictionary
+        :param db: name of the db
+        :param table: name of the table in the db
+        """
+        super(BBRMgr, self).__init__(
+            common_objs,
+            [("CONFIG_DB", swsscommon.CFG_DEVICE_METADATA_TABLE_NAME, "localhost/bgp_asn"),],
+            db,
+            table,
+        )
+        self.enabled = False
+        self.bbr_enabled_pgs = {}
+        self.directory.put(self.db_name, self.table_name, 'status', "disabled")
+        self.__init()
+
+    def set_handler(self, key, data):
+        """ Implementation of 'SET' command for this class """
+        if not self.enabled:
+            log_info("BBRMgr::BBR is disabled. Drop the request")
+            return True
+        if not self.__set_validation(key, data):
+            return True
+        cmds = self.__set_prepare_config(data['status'])
+        rv = self.cfg_mgr.push_list(cmds)
+        if not rv:
+            log_crit("BBRMgr::can't apply configuration")
+            return True
+        self.__restart_peers()
+        return True
+
+    def del_handler(self, key):
+        """ Implementation of 'DEL' command for this class """
+        log_err("The '%s' table shouldn't be removed from the db" % self.table_name)
+
+    def __init(self):
+        """ Initialize BBRNgr. Extracted from constructor """
+        if not 'bgp' in self.constants:
+            log_err("BBRMgr::Disabled: 'bgp' key is not found in constants")
+            return
+        if 'bbr' in self.constants['bgp'] and \
+                'enabled' in self.constants['bgp']['bbr'] and \
+                self.constants['bgp']['bbr']['enabled']:
+            self.bbr_enabled_pgs = self.__read_pgs()
```

Comment:
"BBR is adding `neighbor PEER_GROUP allowas-in 1' for all BGP peer-groups which points to T0"

can you point me code where you identify if a peer group points to T0 or not?

Reply:
Sure. I use file [constants](https://github.com/Azure/sonic-buildimage/pull/5626/files#diff-e6f2fe13a6c276dc2f3b27a5bef79886f9c103194be4fcb28ce57375edf2c23cR39)